src/k8s/pkg/utils/vals/pointer.go

```diff
@@ -0,0 +1,13 @@
+package vals
```

Comment:
better move this under `utils`: `utils.Pointer` and `utils.OptionalBool` are fine as names